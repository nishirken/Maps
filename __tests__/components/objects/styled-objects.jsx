import StyledObjects from 'Components/objects/styled-objects';
import 'jest-styled-components';

describe('Styled objects', () => {
    const TestStyledObjects = shallow(<StyledObjects />);

    it('Rendered', () => {
        expect(TestStyledObjects).toMatchStyledComponentsSnapshot();
    });

    it('Should change right style property, when opened', () => {
        TestStyledObjects.setProps({ open: true });
        expect(TestStyledObjects).toMatchStyledComponentsSnapshot();
    });
});
