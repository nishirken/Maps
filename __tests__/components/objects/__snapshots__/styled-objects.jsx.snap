// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled objects Rendered 1`] = `
.dbdnba {
  height: 100%;
  background: #CFD8DC;
  font-size: 1rem;
  color: #795548;
  position: absolute;
  top: 0;
  right: 0;
  z-index: -1;
  width: 100%;
  overflow-y: auto;
  max-height: 300px;
  min-height: 100%;
  height: auto;
  transition: right 0.4s ease-in-out;
  -webkit-transition: right 0.4s ease-in-out;
}

<div
  className="dbdnba"
/>
`;

exports[`Styled objects Should change right style property, when opened 1`] = `
.eyQMxy {
  height: 100%;
  background: #CFD8DC;
  font-size: 1rem;
  color: #795548;
  position: absolute;
  top: 0;
  right: 100%;
  z-index: -1;
  width: 100%;
  overflow-y: auto;
  max-height: 300px;
  min-height: 100%;
  height: auto;
  transition: right 0.4s ease-in-out;
  -webkit-transition: right 0.4s ease-in-out;
}

<div
  className="eyQMxy"
  open={true}
/>
`;
